include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -40,6 +40,17 @@ struct TorchLoweringPipelineOptions
   Option<bool> decompose{*this, "decompose-complex-ops",
                          llvm::cl::desc("Decompose complex operations."),
                          llvm::cl::init(true)};
+  Option<bool> enableStaticShape{
```

Comment:
Can you put this in a separate struct MhloBackendPipelineOptions and put it in TorchConversion/Transforms/Passes.h

Reply:
I think this might be needed by another pipeline, since static shape and i32 index is not specific options to MHLO?

Reply:
TorchLoweringPipelineOptions is only for the frontend. It should not be used after the backend contract. (after you submit this I will remove it from the other backend pipelines)

Also these options are specific to MHLO. For example, TCP and Linalg uses `index` instead of i64/i32 and the static shape case is always a special case of dynamic shape, so TCP/Linalg is not irregular in that way. Each backend pipeline can have its own options to deal with its specific situation.

cc @burmako for visbility on MHLO design issues that we want to fix for StableHLO.

Reply:
Done